### PR TITLE
fix(iOS): always show native search bar

### PR DESCRIPTION
**Pull request by KatvonRivia**

### CODE REVIEW

File: src/scripts/components/stories/pilot-story/components/chapter-conclusion/chapter-conclusion.module.styl [hunk 1]
```diff
@@ -1,14 +1,13 @@
 @require '../../../../../../variables.styl'
 
 .conclusion
-  height: 100vh
+  height: calc(100svh - 75px)
```
Comment (andreashelms) [inline src/scripts/components/stories/pilot-story/components/chapter-conclusion/chapter-conclusion.module.styl:4]: What is that `-75px`? The search bar height? Maybe put it into a variable or add a comment.

File: src/scripts/components/stories/pilot-story/components/chapter-video/chapter-video.module.styl [hunk 2]
```diff
@@ -49,7 +49,7 @@
 
 @media screen and (min-width: 768px)
   .chapterVideo
-    height: 100vh
+    height: calc(100svh - 75px)
```
Comment (andreashelms) [inline src/scripts/components/stories/pilot-story/components/chapter-video/chapter-video.module.styl:52]: Same here

File: src/scripts/components/stories/pilot-story/components/pilot-carousel/pilot-carousel.tsx [hunk 3]
```diff
@@ -10,6 +10,7 @@ import {Parallax} from 'react-scroll-parallax';
 import Legend, {LegendItems} from '../legend/legend';
 
 import styles from './pilot-carousel.module.styl';
+import SnapWrapper from '../snap-wrapper/snap-wrapper';
```
Comment (andreashelms) [inline src/scripts/components/stories/pilot-story/components/pilot-carousel/pilot-carousel.tsx:13]:
```suggestion
import SnapWrapper from '../snap-wrapper/snap-wrapper';

import styles from './pilot-carousel.module.styl';
```

File: src/scripts/components/stories/pilot-story/components/satellite-carousel/satellite-carousel.module.styl [hunk 4]
```diff
@@ -4,15 +4,17 @@
   --curve-width: 25vw
   position: relative
   width: 100vw
-  height: 100vh
-  padding: emCalc(100px) 0
+  height: calc(100svh - 75px)
```
Comment (andreashelms) [inline src/scripts/components/stories/pilot-story/components/satellite-carousel/satellite-carousel.module.styl:7]: same here

File: src/scripts/components/stories/pilot-story/pilot-story.tsx [hunk 5]
```diff
@@ -30,59 +30,68 @@ import {ChapterContextProvider} from './provider/chapter-provider';
...
-            <Share />
-          </Header>
-          {isDesktop && storyStarted && (
-            <ChapterProgressIndication
-              chapters={chapters}
-              className={styles.chapterProgressIndication}
-              gap={48}
-            />
-          )}
+  useEffect(() => {
```
Comment (Immo-Be) [inline src/scripts/components/stories/pilot-story/pilot-story.tsx:37]: did you try the `useLayoutEffect` hook? I might be fit here. Also, please add a comment to explain what's going on :)

File: src/scripts/components/stories/pilot-story/components/chapter-conclusion/chapter-conclusion.module.styl [hunk 6]
```diff
@@ -1,14 +1,14 @@
 @require '../../../../../../variables.styl'
 
 .conclusion
-  height: 100vh
+  --topPadding: 75px
```
[the remark this answers is not in the document]
Comment (KatvonRivia) [reply src/scripts/components/stories/pilot-story/components/chapter-conclusion/chapter-conclusion.module.styl:4]: calc doesn't work with styl variables, and the variables.styl file doesn't work with normal css variables starting with `--`
We also don't have a global css file, so i had to add the variables in every component 😢

[the remark this answers is not in the document]
Comment (andreashelms) [reply src/scripts/components/stories/pilot-story/components/chapter-conclusion/chapter-conclusion.module.styl:4]: Okay. Then maybe add an issue for later refactoring.

**Review by andreashelms — APPROVED** [review]

**Pull request merged**